### PR TITLE
release: v0.8.0

### PR DESCRIPTION
**Pull request by byashimov**
## About this change—what it does

Release v0.8.0

### CODE REVIEW

File: docs/docs/changelog.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+
+**Note2:** It is now recommended to disable webhooks for Kubernetes >=1.25,
+as native [CRD validation rules](https://kubernetes.io/blog/2022/09/23/crd-validation-rules-beta/) are used.
+
+- **Breaking change:** `ip_filter` field is now of `object` type
+- **Breaking change:** Update user configs for following kinds: PostgreSQL, Kafka, KafkaConnect, Redis, Clickhouse, OpenSearch
 - Add CRD validation rules for immutable fields
-- **Breaking change:** `ip_filter` field is now of `object` type.
-- **Breaking change:** Update user configs for following kinds: PostgreSQL, Kafka, Redis, Clickhouse, OpenSearch, KafkaConnect.  
+- Add user config fields validations (enum, minimum, maximum, minLength, etc)
```
Comment (staceysalamon-aiven) [inline docs/docs/changelog.md:16]:
```suggestion
- Add user config field validations (enum, minimum, maximum, minLength, and others)
```

File: docs/docs/changelog.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+- Add `serviceIntegrations` on service types. Only `read_replica` type for now
```
Comment (staceysalamon-aiven) [inline docs/docs/changelog.md:17]:
```suggestion
- Add `serviceIntegrations` on service types. Only the `read_replica` type is available. 
```

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]
Just a few minor suggestions for the changelog.

**Review by Serpentiel — APPROVED** [review]
lgtm

**Review by Serpentiel — APPROVED** [review]

**Review by staceysalamon-aiven — APPROVED** [review]
Changelog looks good.

**Pull request merged**